cli/formatter/container.go

```diff
@@ -0,0 +1,48 @@
+package formatter
+
+import (
+	"github.com/alibaba/pouch/apis/types"
+	"github.com/alibaba/pouch/pkg/utils"
+)
+
+// ContainerHeader is the map to show container head
+var ContainerHeader = map[string]string{
+	"Name":    "Name",
+	"ID":      "ID",
+	"Status":  "Status",
+	"Created": "Created",
+	"Image":   "Image",
+	"Runtime": "Runtime",
+	"Command": "Command",
+	"ImageID": "ImageID",
+	"Labels":  "Labels",
+	// "Mounts":  "Mounts",
```

Comment:
should we add a comment for this？ what's your concern？

Reply:
no need, I can add the mount data.